.release/ci.hcl

```diff
@@ -11,10 +11,7 @@ project "consul" {
     repository = "consul"
     release_branches = [
       "main",
-      "release/1.8.x",
-      "release/1.9.x",
-      "release/1.10.x",
-      "release/1.11.x",
+      "release/1.11.x"
```

Comment:
Wondering if this is a necessary change?

Reply:
This is inconsequential. I made the change to be consistent with the other branches.